core-plugins/pom.xml

```diff
@@ -90,7 +90,6 @@
     <dependency>
       <groupId>org.python</groupId>
       <artifactId>jython-standalone</artifactId>
-      <version>2.5.2</version>
```

Comment:
Does this need a version, and should it be 2.7.0? Or, should the dependency in the pom.xml be
```
<jython-standalone.version>2.7.0</jython-standalone.version>